src/z3c/relationfield/README.rst

```diff
@@ -19,7 +19,9 @@ Setup
 
 ``z3c.relationfield.Relation`` is a schema field that can be used to
 express relations. Let's define a schema IItem that uses a relation
-field::
+field:
+
+.. code-block:: python
 
```

Comment:
I'm not a fan of this.

I suppose you did it to get Python syntax highlighting?

What if you drop both the `::` literal block syntax and the `.. code-block:: python` directive and use [native doctest blocks](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#doctest-blocks) -- will you get syntax highlighting that way?